tests/scenarios/subscriptions.spec.ts

```diff
@@ -824,11 +826,10 @@ export function testSubscriptionScenarios(): void {
 
         // wait for messages to emit and handler to process
         await Time.minimalSleep();
-        expect(imageMessages.length).to.equal(3);
+        expect(imageMessages.length).to.equal(2);
```

Comment:
Nit: I think it would be useful to either remove the delete of the image completely from this test, or add a comment somewhere here on why it isn't showing up in the subscription.

I know we have it in the Q&A but this might be confusing for anyone that's reading this test specifically.

Reply:
Good call! Removed.